src/dolphin/stitching.py

```diff
@@ -241,6 +241,8 @@ def merge_images(
     if out_dtype is not None:
         out_gdal_dtype = gdal.GetDataTypeName(utils.numpy_to_gdal_type(out_dtype))
         args.extend(["-ot", out_gdal_dtype])
+    if target_aligned_pixels:
```

Comment:
both the changes in this file were fixes in PR #257 , could you set them back?

Reply:
You got it. I reverted it